src/zc/zodbrecipes/__init__.py

```diff
@@ -247,7 +248,7 @@ def install(self):
             else:
                 try:
                     port = int(address)
-                except:
+                except:  # noqa: E722
```

Comment:
```suggestion
                except ValueError:
```

Reply:
So, I trust you scanned the context... as `int(None)` would throw a `TypeError` ...